spec/controllers/public_pages_controller_spec.rb

```diff
@@ -7,22 +7,37 @@
     context "in production" do
       before do
         allow(Rails).to receive(:env).and_return("production".inquiry)
+        allow_any_instance_of(PublicPagesHelper).to receive(:enable_online_intake?).and_return(true)
       end
 
-      it "does not link to the first questions page" do
+      it "includes GA script in html" do
         get :home
 
-        expect(response.body).not_to include "Get started"
-        expect(response.body).not_to include question_path(QuestionNavigation.first)
+        expect(response.body).to include "https://www.googletagmanager.com/gtag/js?id=UA-156157414-1"
       end
 
-      it "includes GA script in html" do
+      it "links to the first question path for digital intake" do
         get :home
 
-        expect(response.body).to include "https://www.googletagmanager.com/gtag/js?id=UA-156157414-1"
+        expect(response.body).to include "Get started"
+        expect(response.body).to include question_path(QuestionNavigation.first)
+      end
+
+      context "when online intake is not enabled" do
+        before do
+          allow_any_instance_of(PublicPagesHelper).to receive(:enable_online_intake?).and_return(false)
+        end
+
+        it "does not link to the first questions page" do
+          get :home
+
+          expect(response.body).not_to include "Get started"
```

Comment:
should we also check that a link to location search is present?

Reply:
cool yeah, I'll beef these up a bit. That would have made me realize there was a second button.